.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-[Things to note: Pull Requests **must** fix an issue. Discussion about the feature / bug takes place in the issue, discussion of the implementation takes place in the PR. Please also see the [Contributing Guide](https://github.com/smartdevicelink/sdl_core/blob/master/.github/CONTRIBUTING.md) for information on branch naming and the CLA.
+[Things to note: Pull Requests **must** fix an issue. Discussion about the feature / bug takes place in the issue, discussion of the implementation takes place in the PR. Please also see the [Contributing Guide](https://github.com/smartdevicelink/sdl_core/blob/master/.github/CONTRIBUTING.md) for information on branch naming and the CLA, and the [SmartDeviceLink GitHub Best Practices](https://d83tozu1c8tt6.cloudfront.net/media/resources/SDL_GitHub_BestPractices.pdf) document for more information on how to enter a pull request.  Once this PR is ready for review, please request one from @jordynmackool.
```

Comment:
> Once this PR is ready for review, please request one from @jordynmackool.
Is this relevant to this repository, or is this just left over from the PR template for `sdl_evolution`?

Reply:
We had discussed making this a part of the PR submission process, with Jordyn fielding PR requests to then triage to the Core team, so that the Core team can focus on the tasks in the current sprint.  If the Core team is still okay with this, let's keep this sentence.  Although at this point, they should request a review from me and not Jordyn, so please update that tag.